string div_layer
float32 Robottime
float32 Cloudtime
float32 Communicationtime